core/modules/dom/src/block.lua

```diff
@@ -0,0 +1,54 @@
+local State = require('state')
+local Type = require('type')
+
+local dom = select(1, ...)
+
+local Block = Type.declare('Block', State)
+
+
+---
+-- Instantiate a new block configuration helper.
+--
+-- @param state
+--    The block configuration state.
+-- @returns
+--    The new block helper instance.
+---
+
+function Block.new(state)
+    local blk = Type.assign(Block, state)
+
+    local name = state.blocks[1]
+    blk.name = name
+
+    return blk
+end
+
+
+---
+-- Retreive the configurations contained by this block.
```

Comment:
```suggestion
-- Retrieve the configurations contained by this block.
```
Typo done at several places.